_docs/install-and-upgrade.md

```diff
@@ -211,7 +212,6 @@ configuration to get up and running:
 
 {:.app-stores}
 [![Bytesized Hosting logo](/img/logos/bytesized-hosting.svg)](https://bytesized-hosting.com/ "Bytesized Hosting")
-[![Cloudron logo](/img/logos/cloudron.svg)](https://cloudron.io/button/io.github.thelounge.html "Cloudron")
```

Comment:
Can you delete the image file too please.

Reply:
@xPaw done